testsuite/tests/contrib/lkt_semantic/test.py

```diff
@@ -0,0 +1,42 @@
+"""
+This tests that the Langkit based python parser can be compiled and can parse a
+simple python program without errors.
+
+TODO: This is hackish and minimalistic. Since Langkit's python parser is a
+fully fledged Langkit generated lib, it would make sense to reuse Libadalang's
+parser test driver.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+from os import path as P
+import subprocess
+import sys
+
+
+LKT_LIB_DIR = P.join(os.environ['LANGKIT_ROOT_DIR'], 'contrib', 'lkt')
+
+o = subprocess.check_output(
+    [sys.executable, P.join(LKT_LIB_DIR, 'manage.py'), 'make', '-P'],
+    # CWD is the lib's directory so that lib is generated in the lib's dir, not
+    # in the test dir.
+    cwd=LKT_LIB_DIR
+)
+
+env = json.loads(subprocess.check_output(
+    [sys.executable, P.join(LKT_LIB_DIR, 'manage.py'), 'setenv', '--json'],
+    # CWD is the lib's directory so that setenv's paths are from the the lib's
+    # dir, not in the test dir.
+    cwd=LKT_LIB_DIR
+))
```

Comment:
Liblktlang is now a Langkit dependency, so you don’t need to build it nor do the setenv. Could you remove this?